catalyst/dl/callbacks/logging.py

```diff
@@ -163,4 +173,17 @@ def on_loader_end(self, state: RunnerState):
             )
 
 
-__all__ = ["VerboseLogger", "ConsoleLogger", "TensorboardLogger"]
+class RaiseExceptionCallback(Callback):
```

Comment:
because it is in `logging.py`, maybe we should rename it to `ExceptionLogger`?

Reply:
It's complicated because it doesn't log anything. It is just re-raise any exception if it needed